stable/rbac-manager/values.yaml

```diff
@@ -23,3 +23,13 @@ affinity: {}
 
 podAnnotations: {}
 podLabels: {}
+
+service:
+  enabled: false
+  annotations: {}
+
+serviceMonitor:
+  enabled: false
+  annotations: ""
```

Comment:
These annotations don't seem to get applied to the serviceMonitor, only the service. Is that intended? If so, maybe it should be a different key?